### PR TITLE
Adapted integration test for Dynatrace Oneagent

### PR DESCRIPTION
**Pull request by arthfl**
Thanks for contributing to the buildpack. To speed up the process of reviewing your pull request please provide us with:

* A short explanation of the proposed change:
Adapted the integration test for Dynatrace Oneagent.
The last two steps wheren't fitting in the overall "concept" and this seemed to have caused problems during test runs

* An explanation of the use cases your change solves
This should solve the problems in the integration test pipelines like https://buildpacks.ci.cf-app.com/teams/main/pipelines/php-buildpack/jobs/specs-lts-integration-develop/builds/981#L62793feb:10533

* [x] I have viewed signed and have submitted the Contributor License Agreement

* [x] I have made this pull request to the `develop` branch

* [x] I have added an integration test

### CODE REVIEW

File: extensions/dynatrace/extension.py [hunk 1]
```diff
@@ -188,6 +190,136 @@ def adding_ld_preload_settings(self):
...
+                self._ctx['BUILD_DIR'],
+                'dynatrace',
+                'oneagent',
+                'agent',
+                'conf',
+                'ruxitagentproc.conf')
+            agent_config_file = open(agent_config_path, 'r')
+            agent_config_data = agent_config_file.readlines()
+            agent_config_file.close()
+        except IOERROR as err:
```
Comment (arjun024) [inline extensions/dynatrace/extension.py:249]: `IOError`?

**Comment by arthfl** [comment]
@arjun024 
I'd really like to run the tests again on our our CFs, but i can't since the wrapper script seems to depend on a rather deprecated (since 2019) Ruby version:
```bash
$ ./scripts/integration.sh 
Your Ruby version is 3.0.4, but your Gemfile specified ~> 2.3
```

How do you folks run them?

**Comment by arjun024** [comment]
@arthfl Could you try using the docker image `cfbuildpacks/ci`? This is the image used by the Buildpacks concourse CI to run the tests.
You can see the concourse task [here](https://github.com/cloudfoundry/buildpacks-ci/tree/master/tasks/run-buildpack-integration-specs) which is for example run in the CI jobs like [this](https://buildpacks.ci.cf-app.com/teams/main/pipelines/php-buildpack/jobs/specs-edge-integration-develop/builds/1014)

Something like this:
```
docker run --init -it -v $PWD:/php-buildpack cfbuildpacks/ci bash
root@62c:/# cd php-buildpack
root@62c:/# <login to your CF>
root@62c:/# ./scripts/integration.sh
```

Specifically for php integration tests, additionally you might need to set a composer github oauth token which you should be able to to set to any random value.

`export COMPOSER_GITHUB_OAUTH_TOKEN=123`

Hope that helps

**Comment by arthfl** [comment]
Thanks for that.
Tried it but it resulted in:
```bash
ginkgo: /lib/x86_64-linux-gnu/libc.so.6: ve[redacted LS_COLORS]ion `GLIBC_2.32' not found (required by ginkgo)
ginkgo: /lib/x86_64-linux-gnu/libc.so.6: ve[redacted LS_COLORS]ion `GLIBC_2.34' not found (required by ginkgo)
```

:-/

**Pull request merged**